tests/setup/global-setup.ts

```diff
@@ -12,7 +12,7 @@ export async function setup() {
 	if (databaseExists) return
 
 	await execaCommand(
-		'prisma migrate reset --force --skip-seed --skip-generate',
+		'npx prisma migrate reset --force --skip-seed --skip-generate',
```

Comment:
`execa` actually handles finding `prisma` automatically. Did you have trouble with this?